test/integration/integration_suite_test.go

```diff
@@ -188,7 +185,9 @@ var _ = BeforeSuite(func(done Done) {
 	}
 
 	if len(k8sClients) == 1 {
-		k8sClient = k8sClients[clusters[0]]
```

Comment:
This can be simplified, since there is only one cluster.

Reply:
fixed